packages/viz/src/Chart/Chart.d.ts

```diff
@@ -3,6 +3,17 @@ import { HvPlotProps } from "./Plot";
 
 export type HvChartClassKey = "root";
 
+type TooltipElementObject = {
+  color: string;
+  name: string;
+  value: number | string;
+};
+
+type TooltipDataObject = {
+  title: string;
+  elements: TooltipElementObject[];
+};
```

Comment:
Export both types, so people can use it if needed

Reply:
fixed on both cases 😄